admin_guide/sdn_troubleshooting.adoc

```diff
@@ -398,9 +397,9 @@ Check that `sysctl net.ipv4.ip_forward` is set to 1 (and check the host if this
 ==== Is `firewalld` Disabled?
```

Comment:
I think we can just remove this section.  Or at least tell them how to see if firewalld is being used and point them elsewhere for docs.